src/components/node-list/node-list-items.js

```diff
@@ -10,80 +10,86 @@ import InvisibleIcon from '../icons/invisible';
 import { arrayToObject } from '../../utils';
 const { escapeRegExp, getHighlightedText } = utils;
 
+export const isTagType = (type) => type === 'tag';
+export const isParameterType = (type) => type === 'parameters';
```

Comment:
on cross-referencing the sidebar code this does not seem to be used anymore - looks like we can delete that.